server/sonar-web/src/main/js/apps/projects/components/ProjectCardOverallMeasures.js

```diff
@@ -111,11 +108,11 @@ export default function ProjectCardMeasures({ measures }: Props) {
       </div>
 
       {measures['ncloc'] != null &&
-        <div className="project-card-measure" data-key="ncloc">
+        <div className="project-card-measure pull-right" data-key="ncloc">
           <div className="project-card-measure-inner">
             <div className="project-card-measure-number">
               <span className="spacer-right">
-                <SizeRating value={formatMeasure(measures['ncloc'], 'INT')} />
+                <SizeRating value={parseInt(measures['ncloc'])} />
```

Comment:
should be either `parseInt(measures['ncloc'], 10)` or `Number(measures['ncloc'])`